src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -99,13 +99,11 @@
 
       <div class="k-search-actions">
         <KButton
-          :appearance="isRegExpMode ? 'action-active' : 'outline'"
+          appearance="secondary"
           :aria-pressed="isRegExpMode"
           class="k-regexp-mode-button"
           data-testid="k-code-block-regexp-mode-button"
-          :is-rounded="false"
           size="small"
-          :title="`Use regular expression (${ALT_SHORTCUT_LABEL}+R)`"
```

Comment:
**Question**: Why are the titles being removed here?

Reply:
Hmm sorry for answering your question with a question, but can you tell me what was it used for here?

Reply:
Hover state help text (e.g. hover over this button tells the user how to use this toggled feature) -- this should remain

Reply:
In my understanding we shouldn't be using title attribute as a tooltip, but since it's out of the scope for this I'm adding all titles in this component back for now.

> Use of the title attribute is highly problematic for:
>
> - People using touch-only devices
> - People navigating with keyboards
> - People navigating with assistive technology such as screen readers or magnifiers
> - People experiencing fine motor control impairment
> -  People with cognitive concerns
>
> This is due to inconsistent browser support, compounded by the additional assistive technology parsing of the browser-rendered page. If a tooltip effect is desired, it is better to use a more accessible technique that can be accessed with the above browsing methods.

[source](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title)

Reply:
TBH something like `aria-label` might be better, or actual tooltips on the button; you can add this info to the `KCodeBlock` ticket in the reskin and we can revisit then 👍🏼